war/src/main/js/add-item.js

```diff
@@ -100,14 +100,14 @@ $.when(getItems()).done(function(data) {
       });
     }
 
-    function enableSubmit(status) {
+
+    function enableButton(status) {
       var btn = $('form .footer .btn-decorator button[type=submit]');
       if (status === true) {
         if (btn.hasClass('disabled')) {
           btn.removeClass('disabled');
           btn.prop('disabled', false);
         }
-        btn.focus();
       } else {
         if (!btn.hasClass('disabled')) {
           btn.addClass('disabled');
```

Comment:
I know the above code is not part of this PR, but it could be cleaned up. The `btn.hasClass('disabled')` calls are dumb. I'm sure the above code could be more simply stated as ...

```javascript
function enableButton(status) {
  var btn = $('form .footer .btn-decorator button[type=submit]');

  btn.prop('disabled', !status);
  if (status === true) {
    btn.removeClass('disabled');
  } else {
    btn.addClass('disabled');
  }
}
```